pages/search.js

```diff
@@ -10,7 +10,7 @@ import Badge from "@components/Badge";
 import logger from "@config/logger";
 import Input from "@components/form/Input";
 
-export async function getServerSideProps(context) {
```

Comment:
What is the `_` for?

Reply:
here, the variable `context` was not being used, so I shadowed it using `_`

Reply:
this is the best coding practice, right?

Reply:
I would remove the variable if not used

Reply:
> I would remove the variable if not used

May I?

Reply:
I don't think adding a `_` is best practice

Reply:
> I don't think adding a `_` is best practice

Got it, thank you for correcting my misunderstanding. Removing the `_` from the code.